crates/revm/revm-inspectors/Cargo.toml

```diff
@@ -14,5 +14,6 @@ reth-rpc-types = { path = "../../rpc/rpc-types" }
 revm = { version = "3" }
 # remove from reth and reexport from revm
 hashbrown = "0.13"
+ethers-core = { git = "https://github.com/gakonst/ethers-rs", features = ["eip712"] }
```

Comment:
why do we need this, we should not need this

Reply:
Will remove it. I had added it to import CallLogFrame but its already available